src/components/AboutSection/AboutSection.tsx

```diff
@@ -0,0 +1,36 @@
+import { Product } from '../../types/Product';
+import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { getProduct } from '../../utils/fetchData';
+import './AboutSection.scss';
+
+export const AboutSection = () => {
+  const [good, setGood] = useState<Product | null>(null);
+  const { productId } = useParams();
+
+  useEffect(() => {
+    if (productId) {
+      getProduct('./api/phones.json', productId)
+        .then(product => setGood(product))
+        .catch(error => console.log(error));
+    }
+  }, []);
```

Comment:
There is no need to fetch product in each component.
Fetch product at ProductPageLayout and pass it as a prop